README.md

```diff
@@ -42,25 +42,57 @@ pagination, limiting by `batchSize` and ordering by `orderingColumn`.
 | `dsn`            | [DSN](https://en.wikipedia.org/wiki/Data_source_name) to connect to Redshift.                                                                                              | **true** | `postgres://username:password@endpoint:5439/database` |               |
-| `table`          | Name of the table from which the connector reads from.                                                                                                                     | **true** | `table_name`                                          |               |
-| `orderingColumn` | Column used to order the rows. <br /> Keep in mind that the data will be sorted by this column, so the column must contain unique, consistent values suitable for sorting. | **true** | `id`                                                  |               |
+| `tables.*.orderingColumn`                  | Column used to order the rows. <br /> Keep in mind that the data will be sorted by this column, so the column must contain unique, consistent values suitable for sorting.    | true     |               |
+| `tables.*.keyColumns`                  | Comma-separated list of column names to build the sdk.Record.Key. See more: [Key handling](#key-handling).   | false     |               |
+| ~~`table`~~          | Name of the table from which the connector reads from. **Deprecated: use `tables` instead.**                                                                                                                     | **false** | `table_name`                                          |               |
+| ~~`orderingColumn`~~ | Column used to order the rows. <br /> Keep in mind that the data will be sorted by this column, so the column must contain unique, consistent values suitable for sorting. **Deprecated: use `tables.*.orderingColumn` instead.** | **false** | `id`                                                  |               |
 | `snapshot`       | Whether the connector will take a snapshot of the entire table before starting cdc mode.                                                                                   | false    | `false`                                               | "true"        |
-| `keyColumns`     | Comma-separated list of column names to build the `sdk.Record.Key`. See more: [Key handling](#key-handling).                                                               | false    | `id,name`                                             |               |
+| ~~`keyColumns`~~     | Comma-separated list of column names to build the `sdk.Record.Key`. **Deprecated: use `tables.*.keyColumns` instead.**                                                               | false    | `id,name`                                             |               |
```

Comment:
can we please move all the deprecated params to the end of the table

Reply:
done